### PR TITLE
Adjust backup screen

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot from 2023-07-04 16-12-18

fixes : https://github.com/massalabs/station-massa-wallet/issues/450#issuecomment-1620219930

### CODE REVIEW

File: wails-frontend/src/pages/backupKeyPairs.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ function BackupKeyPairs() {
 
   return (
     <Layout>
-      <form ref={form} onSubmit={handleSubmit}>
+      <form ref={form} onSubmit={handleSubmit} className="max-w-xs">
```
Comment (maxime-carabina) [inline wails-frontend/src/pages/backupKeyPairs.tsx:99]: Can you send me a picture of before result please ? 🤔

Comment (Thykof) [reply wails-frontend/src/pages/backupKeyPairs.tsx:99]: git checkout and test by yourself ;) it's the best way to make the product reliable!

Comment (maxime-carabina) [reply wails-frontend/src/pages/backupKeyPairs.tsx:99]: > @maxime-carabina check the comment linked ins the description

Thank you! It's okay for me 😉

**Review by Thykof — APPROVED** [review]
LGTM

**Review by Pierregvx — APPROVED** [review]

**Pull request merged**